ext/groonga/extconf.rb

```diff
@@ -169,6 +169,7 @@ def configure_command_line(prefix)
   command_line << "--disable-document"
   command_line << "--disable-benchmark"
   command_line << "--without-cutter"
+  command_line << "--disable-groonga-httpd" if ENV['WITHOUT_HTTPD']
```

Comment:
I suggest that we add a prefix "RROONGA" for unifying the naming with other environmental variables.